spec/atom-protocol-handler-spec.coffee

```diff
@@ -1,15 +1,23 @@
 {$} = require '../src/space-pen-extensions'
 
+fetch = (url, callback) ->
```

Comment:
We actually have `window.fetch` natively now - what do you think about using that now that we've moved away from jQuery in core?